### PR TITLE
[loki] feat: support existing alerting group configmap 

### PR DESCRIPTION
**Pull request by deimosfr**

### CODE REVIEW

**Review by benjaminch — DISMISSED** [review]

**Comment by zanhsieh** [comment]
@deimosfr 
Please sign DCO.

**Comment by deimosfr** [comment]
> @deimosfr Please sign DCO.

Done, thx

**Comment by zanhsieh** [comment]
@deimosfr 
Please bump the chart version to latest too. Thank you.

**Comment by deimosfr** [comment]
> @deimosfr Please bump the chart version to latest too. Thank you.

done thx

**Comment by deimosfr** [comment]
> @deimosfr Please bump the chart version to latest too. Thank you.

Everything should be ok now, hope it will pass tests and be merged :)

**Review by zanhsieh — APPROVED** [review]

File: charts/loki/values.yaml [hunk 1]
```diff
@@ -318,3 +318,7 @@ alerting_groups: []
 #    - alert: HighThroughputLogStreams
 #      expr: sum by(container) (rate({job=~"loki-dev/.*"}[1m])) > 1000
 #      for: 2m
+
+useExistingAlertingGroup:
+  enabled: false
+  configmapName: ""
```
Comment (zanhsieh) [inline charts/loki/values.yaml:324]: Check failure on line 324 in charts/loki/values.yaml

GitHub Actions
/ lint-test

charts/loki/values.yaml#L324

[new-line-at-end-of-file] no new line character at the end of file

**Review by zanhsieh — CHANGES_REQUESTED** [review]

File: charts/loki/values.yaml (hunk 1 above)
Comment (deimosfr) [reply charts/loki/values.yaml]: done

**Review by zanhsieh — APPROVED** [review]

**Pull request merged**